src/routes/contactus/+page.svelte

```diff
@@ -5,85 +5,103 @@
 	const avatarStyle = 'w-20 rounded-full';
 </script>
 
-<div>
-	<Heading tag="h1" class="mb-4" customSize="text-4xl font-extrabold  md:text-5xl lg:text-5xl"
-		>Contact Us</Heading
-	>
-	<div class="text-base space-y-4">
-		<p>No matter who you are, we want you to join us.</p>
-		<p>
-			As far as college clubs go, ACM is a truly special one. Our club focuses on all things
-			technology and computers, however, you don't need to be good with computers to enjoy our club.
-		</p>
-		<p>If you have any questions feel free to contact any of us.</p>
-	</div>
-
-	<div class="w-fit mx-auto pt-4 space-y-4">
-		<Heading tag="h1" customSize="text-xl font-extrabold  md:text-2xl lg:text-3xl"
-			>2024 Officer Team</Heading
-		>
-		<div class="flex items-center">
-			<Avatar src="{base}/images/officers/dakotaG.jpg" alt="Dakota Gullicksen" class={avatarStyle} />
-			<div class="pl-2">
-				<p class="font-bold">Dakota Gullicksen - President</p>
-				<a href="mailto:kogull01@wsc.edu">kogull01@wsc.edu</a>
-			</div>
-		</div>
-
-		<div class="flex items-center">
-			<Avatar src="{base}/images/officers/JaydenC.jpg" alt="Jayden Carper" class={avatarStyle} />
-			<div class="pl-2">
-				<p class="font-bold">Jayden Carper - Vice President</p>
-				<a href="mailto:jacarp01@wsc.edu">jacarp01@wsc.edu</a>
-			</div>
-		</div>
 
-		<div class="flex items-center">
-			<Avatar src="{base}/images/officers/Aunik.jpg" alt="Ajharul Islam Aunik" class={avatarStyle} />
-			<div class="pl-2">
-				<p class="font-bold">Ajharul Islam Aunik - Secretary</p>
-				<a href="mailto:ajauni01@wsc.edu">ajauni01@wsc.edu</a>
-			</div>
+<div class="bg-white py-8">
+	<div class="mx-auto max-w-7xl gap-x-8 gap-y-20 px-6 lg:px-8">
+		<div class="max-w-2xl">
+			<h2 class="text-3xl font-bold text-gray-900 sm:text-4xl">Contact Us</h2>
+			<p class="mt-6 text-lg pl-6 leading-8">
+				No matter who you are, we want you to join us.
+				As far as college clubs go, ACM is a truly special one. Our club focuses on all things
+				technology and computers, however, you don't need to be good with computers to join our club.
+				If you have any questions feel free to contact any of us.
+			</p>
 		</div>
 
-		<div class="flex items-center">
-			<Avatar src="{base}/images/officers/AdahH.jpg" alt="Adah Heng" class={avatarStyle} />
-			<div class="pl-2">
-				<p class="font-bold">Adah Heng - Treasurer</p>
-				<a href="mailto:adheng01@wsc.edu">adheng01@wsc.edu</a>
-			</div>
-		</div>
+		<p class="pt-20 font-bold text-xl md:text-3xl">2024 Ofiicer Team</p>
 
-		<div class="flex items-center">
-			<Avatar src="{base}/images/officers/no-profile.png" alt="Tristin Patzel" class={avatarStyle} />
-			<div class="pl-2">
-				<p class="font-bold">Tristin Patzel - Historian</p>
-				<a href="mailto:trpatz01@wsc.edu">trpatz01@wsc.edu</a>
-			</div>
-		</div>
+		<ol class="py-4 px-4 grid gap-x-8 gap-y-12 md:grid-cols-2 lg:grid-cols-3 sm:gap-y-16">
```

Comment:
I like the style and look of the cards but there's a way we can make this code a lot smaller by making a component that we can reuse. Look at how to make components or we can talk and discuss how to make components. Components allow you to make a custom element you can use anywhere. You can make a custom component have custom parameters so we can pass information to it without repeating code. https://svelte.dev/docs/svelte-components

Reply:
Also Brandon and Tristen are no longer officers so don't include them

Reply:
I have made all the made all the changes you suggested.